src/winetricks

```diff
@@ -2887,6 +2923,7 @@ w_workaround_wine_bug()
         da*) w_warn "Arbejder uden om wine-fejl ${1} ${_W_msg}" ;;
         de*) w_warn "Wine-Fehler ${1} wird umgegangen ${_W_msg}" ;;
         pl*) w_warn "Obchodzenie błędu w wine ${1} ${_W_msg}" ;;
+        pt*)   w_warn "Trabalhando em torno do bug do wine ${1} ${_W_msg}" ;;
```

Comment:
Please remove the extra spaces before `w_warn`.

Reply:
Ok